packages/terra-functional-testing/src/commands/utils/describeViewports.js

```diff
@@ -0,0 +1,40 @@
+const setViewport = require('./setViewport');
+
+/**
+ * Mocha `describe` block to run tests through a defined list of viewports.
```

Comment:
Is this still mocha?

Reply:
We're still using the mocha testing framework here, but we use jest (expect) as the assertion library